drivers/include/net/netdev2.h

```diff
@@ -10,15 +10,153 @@
  */
 
 /**
- * @defgroup    drivers_netdev_netdev2 Generic Low-Level Network Driver Interface v2
+ * @defgroup    drivers_netdev_netdev2 Network Device Driver Interface
  * @ingroup     drivers_netdev
  * @brief       This is the second version of a generic low-level network
  *              driver interface
  * @{
  *
- * This interface is supposed to be a low-level interface for network drivers.
+ * @section sec_about About
```

Comment:
but e.g. not like `sys/arduino/doc.txt` :-)

But simple: I can't see anything in the coding conventions, that tells me to use Markdown syntax for doxygen blocks -> so I refuse to change it right now. But if you put this into the coding conventions (an maybe open an issue for tracking the adaption of 'old' style doc), I will be happy to adapt this PR as well! :-)